src/Infrastructure/Infrastructure.csproj

```diff
@@ -13,12 +13,12 @@
         <PackageReference Include="IdentityServer4.AspNetIdentity" Version="3.1.2" />
     </ItemGroup>
 
-    <ItemGroup>
-        <ProjectReference Include="../Domain/Domain.csproj" />
-        <ProjectReference Include="..\Application\Application.csproj" />
```

Comment:
Should be / for consistency

Reply:
Can you suggest? I do not have a development environment setup on Windows.

Reply:
6233f0f7372b2450a4e8913e2939f7ea8616ca24